teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher
+  attr_accessor :specialization, :age, :name
+
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```

Comment:
**OPTIONAL** - According to the project requirements, this class should Inherit from the Person class. Please fix this 🙏